docs/tutorials/getting_started/index.rst

```diff
@@ -142,19 +142,25 @@ Powershell Prompt.  Regardless of OS and shell, the following steps are the same
 2. Run the :doc:`idaes get-extensions command<../../../reference_guides/commands/get_extensions>`
    to install the compiled binaries. These binaries include solvers and function libraries.
    See :ref:`Binary Packages <tutorials/getting_started/binaries:Binary Packages>`
-   for more details.
+   for more details.  For example::
 
-    idaes get-extensions
+      idaes get-extensions
 
-..
+   ..
+
+   or::
+
+     idaes get-extensions --extra petsc
```

Comment:
Now that the text about PETSc has been removed, it would probably be helpful to include something here about when to (or not to) include `--extras petsc`.

Reply:
Okay added that and a link to the more detailed PETSc docs.